lang/system/pl.xml

```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godz.</clock_12_hour>
+				<clock_24_hour>Zegar 24-godz.</clock_24_hour>
 			</time_format>
 		</date_time>
 		<language name="Język">
 			<system_language>Język systemu</system_language>
-			<input_language name="język wprowadzania">
-				<keyboards name="Klawiatury">
+			<input_language name="Języki wprowadzania">
+				<keyboards name="Rodzaj klawiatury">
```

Comment:
Revert it, is good, no have two word

Reply:
Done

Reply:
you have not done here
```suggestion
				<keyboards name="Klawiatury">
```

Reply:
@Zangetsu38 here is no problem here, is fine

Reply:
Is already fixed